playbooks/besman-counterfit-0.0.1-playbook.sh

```diff
@@ -125,8 +127,9 @@ function __besman_prepare() {
     export EXECUTION_TIMESTAMP
     
     source ~/.bashrc
-    cp -f $BESMAN_COUNTERFIT_LOCAL_PATH/targets/results/${COUNTERFIT_ATTACKID}/run_summary.json $DETAILED_REPORT_PATH
-    [[ ! -f $DETAILED_REPORT_PATH ]] && __besman_echo_red "Could not find report @ $DETAILED_REPORT_PATH" && return 1
+    [[ ! -d $BESMAN_ASSESSMENT_DATASTORE_DIR/models/$BESMAN_ARTIFACT_NAME/dast ]] && mkdir -p $BESMAN_ASSESSMENT_DATASTORE_DIR/models/$BESMAN_ARTIFACT_NAME/dast
+    cp -f $BESMAN_COUNTERFIT_LOCAL_PATH/counterfit/targets/results/${COUNTERFIT_ATTACKID}/run_summary.json $SUMMARY_REPORT_PATH
```

Comment:
Please make this path `$BESMAN_COUNTERFIT_LOCAL_PATH/counterfit/targets` as a variable instead of hardcoding wherever applicable.

Reply:
Will change it.